function fullOfError() {
  const variable = 2;
  variable++;
  return anotherVariable;
}
